tools/osbuilder/node-builder/azure-linux/README.md

```diff
@@ -90,6 +93,39 @@ popd
 sudo systemctl start tardev-snapshotter
 ```
 
+### Build specific release or debug UVM:
```

Comment:
We are essentially duplicating instructions here. Can we minimize what we add by adding the context directly below `For Kata-CC:` and `For Kata:`?